atk.yml

```diff
@@ -70,6 +70,7 @@ services:
         APT_DEPENDENCIES: "${DEFAULT_APT_DEPENDENCIES}"
         PIP_REQUIREMENTS: "${DEFAULT_PIP_REQUIREMENTS}"
         ROSDEP_METAPACKAGE: "art_art5_meta"
+        ROSDEP_SKIP_KEYS: "python3-torchvision tf_transformations"
```

Comment:
Explanation: `python3-torchvision` isn't a valid package on aarch64 (the jetson architecture), so we'll ignore it. `tf_transformations` is actually not a valid package name, but the nmea (idk what that is; maybe gps) driver has it as an `exec_depend`. It would run find, but i didn't like the warning.

Reply:
- nmea is the message standard for gps. This is fine - if we need the driver ew can just remove `tf_transformations` from here right?
- Confused why tensorflow (`tf_transformations`) would work fine while pytorch doesn't?
- What do you mean by "we'll ignore it"? Does this mean we can't run cone detection on the jetson anymore? Does this mean we can't run it with gpu support?

Reply:
- I mean `tf_transformations` are specified in the `exec_depend` of the nmea package, but that's not an actual package name. It's a mistake in their `package.xml` file, which is a submodule of this repo and an exec depend itself of the `art_art5_meta` package.
- waht do you mean? pytorch works? `tf_transformations` just isn't a valid package name
- no we can run everything fine, will probably need to specify `tf_transformations` in the pip requirements for that service